packages/core-settings/package.js

```diff
@@ -0,0 +1,28 @@
+Package.describe({
+  name: 'unchained:core-settings',
+  version: '0.0.1',
```

Comment:
Please use same version as other packages, in the beginning we just publish all packages together with search & replace